node/node.go

```diff
@@ -1997,6 +1997,7 @@ func (app *App) startAPIServices(ctx context.Context) error {
 			logger.Zap().Named("JSON"),
 			app.Config.API.JSONListener,
 			app.Config.API.JSONCorsAllowedOrigins,
+			app.Config.API.JSONCorsEverywhere,
```

Comment:
I think you can just pass `app.Config.API` as argument to `NewJSONHTTPServer`. You also need to adjust the code in tests for the changed function signature.

Reply:
I'm not going to do "proper" refactorings in that PoC :) Most of that JSON server stuff is IMO to be completely refactored for the final solution.